src/tools/r2rdump/Amd64/GcInfo.cs

```diff
@@ -186,7 +187,15 @@ public GcInfo(byte[] image, int offset, Machine machine, ushort majorVersion)
 
             SlotTable = new GcSlotTable(image, machine, _gcInfoTypes, ref bitOffset);
 
-            Transitions = GetTranstions(image, ref bitOffset);
+            // Try partially interruptible first
+            if (NumSafePoints > 0)
+            {
+                LiveSlotsAtSafepoints = GetLiveSlotsAtSafepoints(image, ref bitOffset);
+            }
+            else
+            {
+                Transitions = GetTransitions(image, ref bitOffset);
```

Comment:
Thanks for fixing the typo, that kept annoying me for quite some time but I never found the energy to send out a PR just to fix a naming typo :-).